pipecat-base/app.py

```diff
@@ -28,26 +27,57 @@
 )
 logger.remove()
 logger.add(sys.stderr, format=session_logger_format)
-logger.configure(extra={"session_id":"NONE"})
+logger.configure(extra={"session_id": "NONE"})
+
 
 @app.post("/bot")
-async def handle_bot_request(body: dict, x_daily_room_url: Annotated[str | None, Header()] = None, x_daily_room_token: Annotated[str | None, Header()] = None, x_daily_session_id: Annotated[str | None, Header()] = None, proxy_connection: Annotated[str | None, Header()] = None):
-    await run_default_bot(body, x_daily_room_url, x_daily_room_token, x_daily_session_id)
+async def handle_bot_request(
+    body: dict,
+    x_daily_room_url: Annotated[str | None, Header()] = None,
+    x_daily_room_token: Annotated[str | None, Header()] = None,
+    x_daily_session_id: Annotated[str | None, Header()] = None,
+    proxy_connection: Annotated[str | None, Header()] = None,
+):
+    launch_default_bot(body, x_daily_room_url, x_daily_room_token, x_daily_session_id)
     return {}
 
-async def run_default_bot(body, x_daily_room_url, x_daily_room_token, x_daily_session_id):
+
+async def default_bot_main(body, x_daily_room_url, x_daily_room_token, x_daily_session_id):
     session_logger = logger.bind(session_id=x_daily_session_id)
-    response = await bot(body, x_daily_room_url, x_daily_room_token, x_daily_session_id, session_logger)
+    response = await bot(
+        body, x_daily_room_url, x_daily_room_token, x_daily_session_id, session_logger
+    )
     return response
 
+
+def default_bot_process(body, x_daily_room_url, x_daily_room_token, x_daily_session_id):
+    # This is a different process so we need to configure the logger again.
+    logger.remove()
+    logger.add(sys.stderr, format=session_logger_format)
+    logger.configure(extra={"session_id": "NONE"})
+
+    asyncio.run(default_bot_main(body, x_daily_room_url, x_daily_room_token, x_daily_session_id))
+
+
+def launch_default_bot(body, x_daily_room_url, x_daily_room_token, x_daily_session_id):
+    process = Process(
+        target=default_bot_process,
+        args=(body, x_daily_room_url, x_daily_room_token, x_daily_session_id),
+    )
+    process.start()
```

Comment:
I think we won't have issues running this way.  We will need to test in a deployed environment to be sure, but the multiprocessing module I used earlier was a `ProcessPoolExecutor`.  It looks like this boils down to `fork()` so I suspect we'll be fine.